x-pack/plugins/observability/server/services/openai/config.ts

```diff
@@ -24,6 +24,12 @@ export const azureOpenAIConfig = schema.object({
 
 export const observabilityCoPilotConfig = schema.object({
   enabled: schema.boolean({ defaultValue: false }),
+  feedback: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+    url: schema.string({
+      defaultValue: `https://0d0uj24psl.execute-api.us-east-1.amazonaws.com/gaifeedback`,
```

Comment:
@SeanHeelan do you mind fielding these questions?

Reply:
Are calls to this URL free? or does it have a pricing model? Are there concerns about a route without authorization racking up a large bill?

Reply:
> I have a couple questions about this hardcoded URL:
> 
> * Is there any chance the backing AWS resource might change and this URL would no longer be the desired endpoint? Would OnPrem users know to override it?
> * Is there any risk of ownership of this URL expiring? There could be a potential risk of someone squatting that URL and we would be sending accessTokens.

The URL should not change (from what I can tell) or expire, as those first characters after the `https://` are an AWS allocated API ID, which should be static, and unique. 

However, this does bring up something we should consider: lets say we want to move away from AWS, or for whatever reason _we_ create a new API endpoint and want to migrate deployments to it. We could handle this by either registering our own domain/subdomain for use here and then we would control where it redirects to. That is a bit of a faff, and I'm unsure who I would even talk to about getting this started. Given I don't anticipate a huge number of users for the AI assistant in 8.9 perhaps a workable solution might be to define this URL as a variable in a configuration file that we, but not the customer, can edit? Would that solve the problem of allowing us to modify it if necessary? 

Thoughts?

Reply:
> Given I don't anticipate a huge number of users for the AI assistant in 8.9 perhaps a workable solution might be to define this URL as a variable in a configuration file that we, but not the customer, can edit? 

Fwiw, his is currently the case on Cloud, as in, only Support can set it; for local environments, people can do whatever they want.

Reply:
How important or valuable is it to have this enabled _by default_ for **on-prem** deployments? Would it be feasible and satisfactory for you if we enable it only in the environments managed by us, using Cloud/Serverless-specific configuration instead of hardcoding it into our distributables? 

In this case, we won't run the risk of having already released versions of Kibana on-prem that will **always** rely on this URL unless explicitly changed by the admin.